### PR TITLE
feat: Improve Empty component

### PR DESCRIPTION
**Pull request by DiamondYuan**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

1. Describe the source of requirement, like related issue link.

https://github.com/ant-design/ant-design/issues/15442

### 💡 Solution

1. How to fix the problem, and list final API implementation and usage sample if that is an new feature.

2. GIF or snapshot should be provided if includes UI/interactive modification.

### 📝 Changelog description

> Describe changes from userside, and list all potential break changes or other risks.

1. English description
empty component support imageSize prop
2. Chinese description (optional)
empty 组件支持设定图片尺寸
### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by DiamondYuan** [comment]
还缺文档

**Review by afc163 — DISMISSED** [review]

File: components/empty/demo/customize.md [hunk 1]
```diff
@@ -7,18 +7,19 @@ title:
 
 ## zh-CN
 
-自定义图片、描述、附属内容。
+自定义图片链接、图片大小、描述、附属内容。
 
 ## en-US
 
-Customize image, description and extra content.
+Customize image source,image size, description and extra content.
```
Comment (imhele) [inline components/empty/demo/customize.md]:
```diff
- source,image size
+ source, image size
```
absence of a space

**Review by afc163 — APPROVED** [review]

**Review by ztplz — APPROVED** [review]

**Comment by zombieJ** [comment]
抱歉看到的有点晚，感觉这些可不可以合到 `image` 属性里？
* `image="http://..."`
* `image={{ simple: true }}`
* `image={{ src: 'xxxxxx', style: {{ height: 300 }} }}`

`image="simple"` 和 `imageSize` 感觉有些散

**Comment by afc163** [comment]
```
image={{ src: 'xxxxxx', style: {{ height: 300 }} }}
```

这个还是推荐这样的语法会比较好：`image={<image src="xxx" />}`。

**Comment by afc163** [comment]
或者

```
<Empty image={Empty.PRESETTED_IMAGE} />
```

**Comment by zombieJ** [comment]
> 这个还是推荐这样的语法会比较好：`image={<image src="xxx" />}`。
> 或者
> `<Empty image={Empty.PRESETTED_IMAGE} />`

👍 可以的
直接判断是不是 ReactNode 并提供一个内置的 image 感觉刚刚好。

@DiamondYuan ，你觉得呢？

**Comment by DiamondYuan** [comment]
> > 这个还是推荐这样的语法会比较好：`image={<image src="xxx" />}`。
> > 或者
> > `<Empty image={Empty.PRESETTED_IMAGE} />`
> 
> 👍 可以的
> 直接判断是不是 ReactNode 并提供一个内置的 image 感觉刚刚好。
> 
> @DiamondYuan ，意下如何？

我觉得这样好。等有空了提交一下。

**Comment by afc163** [comment]
那么是 `PRESETTED_IMAGE_1` `PRESETTED_IMAGE_2` 么？

还是 `PRESETTED_IMAGES[0]` 和 `PRESETTED_IMAGES[1]`。

**Comment by zombieJ** [comment]
* PRESETTED_IMAGE_DEFAULT
* PRESETTED_IMAGE_SIMPLE

这种？

**Comment by afc163** [comment]
可以

**Comment by DiamondYuan** [comment]
> * PRESETTED_IMAGE_DEFAULT

PRESENTED_IMAGE_DEFAULT

PRESENTED_IMAGE_SIMPLE

**Comment by DiamondYuan** [comment]
https://github.com/ant-design/ant-design/pull/15487

转移阵地 Transfer position

**Pull request merged**